pages/1.10/release-notes/1.10.4/index.md

```diff
@@ -0,0 +1,231 @@
+---
+layout: layout.pug
+navigationTitle:  Release Notes for 1.10.4
+title: Release Notes for 1.10.4
+menuWeight: 10
+excerpt:
+---
+
+These are the release notes for DC/OS 1.10.4.
+
+[button color="purple" href="https://downloads.dcos.io/dcos/stable/1.10.2/dcos_generate_config.sh?_ga=2.35647400.804658989.1513010878-1207888710.1504116514"]Download DC/OS Open Source[/button]
+
+# Issues Fixed in DC/OS 1.10.4
+
+- CORE-1375 - Docker executor does not hang due to lost messages.
+- DCOS-1877 - DC/OS CA certificate bundle now gets propagated to public slaves.
+- DOCS-2169	- Updated [ports list](1.10/installing/ent/ports) for DC/OS. [enterprise type="inline" size="small"/]
+- DCOS-19327 - Diagnostics bundles no longer contain sensitive cluster configuration values.
+- DCOS-19399 - Marathon now supports upgrading to JDK 1.8.0_152.
+- DCOS_OSS-1828	- Prometheus plugin now authenticates on master nodes.
+- DCOS_OSS-1898	- DC/OS CLI can now retrieve metrics for Dockerized tasks.
+- DCOS_OSS-1942 - Prometheus plugin now has permissions to access file socket.
+
+# Notable Changes in DC/OS 1.10.4
+
+- Mesos 1.4.1 is integrated with DC/OS 1.10.4.
+- Marathon 1.5.5 is integrated with DCOS 1.10.4.
+- DC/OS Java Developer Kit upgraded to the latest version: JDK 8u152.
+- DC/OS is compatible with the newest Docker version: 17.05.0.
+
+# Issues Fixed in DC/OS 1.10.3
```

Comment:
Hello @pavisandhu  - first question, why are the issues fixed in 1.10.3,  1.10.2 and 1.10.1 in this diff?  Should we be reviewing them?

Shouldn't the patch/ diff be only for 1.10.4?

/cc @sascala

Reply:
@orsenthil - It's our convention to include info. for older patch releases in the RN (so customers have all the info. in one place). You only need to review the info. for 1.10.4, as the info. for 1.10.3, 1.10.2, and 1.10.1 is unchanged.